src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/IndexTests.kt

```diff
@@ -260,4 +260,25 @@ class IndexTests
         assertThat(filterSelect.select("option").map { it.attr("value") }).containsExactly("a", "b")
         assertThat(filterSelect.select("option").map { it.text() }).containsExactly("a", "b")
     }
+
+    @Test
+    fun `it does not render run workflow link if user does not have permission to see it`()
```

Comment:
This test is just the same as `it does not render run report link if user does not have permission to see it` isn't it? Could just rename that tests to `it does not render run report or run workflow link if user does not have permission to see it`